Code/BasicFilters/json/DICOMOrientImageFilter.json

```diff
@@ -0,0 +1,151 @@
+{
+  "name" : "DICOMOrientImageFilter",
+  "template_code_filename" : "ImageFilter",
+  "template_test_filename" : "ImageFilter",
+  "number_of_inputs" : 1,
+  "pixel_types" : "BasicPixelIDTypeList",
```

Comment:
I think this should also be for all pixel types (including label and vector), AllPixelIDTypeList. DICOM can also contain vector/color pixels. Also, if I have a matching segmentation I'd like to apply the filter to both the original DICOM scalar volume and to the label image.

Reply:
We can add vector support. The `BasicPixelIDTypeList` includes the integers which are commonly used for label representation the Label pixel ID refer to the label map representation ( I know it's commonly confused ).